itou/www/approvals_views/forms.py

```diff
@@ -39,9 +42,19 @@ class ApprovalForm(forms.Form):
     status_future = forms.BooleanField(label="PASS IAE valide (non démarré)", required=False)
     status_expired = forms.BooleanField(label="PASS IAE expiré", required=False)
 
-    def __init__(self, siae_pk, *args, **kwargs):
+    EXPIRY_CHOICES = [("1", "Moins d'1 mois"), ("3", "Moins de 3 mois"), ("7", "Moins de 7 mois"), ("0", "Tous")]
+    expiry = forms.ChoiceField(
+        label="Fin du parcours en IAE",
+        choices=EXPIRY_CHOICES,
+        widget=forms.RadioSelect,
+    )
+
+    def __init__(self, siae_pk, data, *args, **kwargs):
         self.siae_pk = siae_pk
-        super().__init__(*args, **kwargs)
+        if not dict(data):
```

Comment:
Pourquoi le `dict()` ? `if not data:` ne suffit pas ?

Reply:
data est un QueryDict() qui est évalué à True même vide :'(

Reply:
```
(Pdb) data
<QueryDict: {}>
(Pdb) bool(data)
False
```
:suspect:

Reply:
Je ne comprends pas comment j'ai fait pour avoir ce problème.... :face_with_spiral_eyes: 

En effet, c'est évalué à False.